students/EricAdams/session06/test_send_thank_you_write_letter_to_file.txt

```diff
@@ -0,0 +1,3 @@
+Jeff Bezos
```

Comment:
are these being generated by the tests? if so, you don't want to put them in git.